src/Pages/Profile/MemberView/ProfileModifier.css

```diff
@@ -0,0 +1,37 @@
+#enclose { 
```

Comment:
Rename this file to use lower case and dashes. An explanation can be found [here](https://www.lifewire.com/naming-css-style-sheet-files-3466881)